incubator/istio/templates/addons/service/servicegraph.yaml

```diff
@@ -1,5 +1,6 @@
+{{- if and .Values.istio.install (not .Release.IsInstall) -}}
```

Comment:
The service graph image should also be updated to `docker.io/istio/servicegraph:0.2.6`

Reply:
Nice catch. Thanks!